### DISCUSSION TITLE
JustPy instead of wxPython?

### DISCUSSION THREAD

**Discussion by ghost** [elimintz/justpy#658] [General]
Hello,
I have a PostgreSQL database for meetings and conferences with around 20 tables and plenty of relations between the tables (people, events, rooms, etc.). So far I have made a GUI with wxPython. Now I want to switch to a WEB GUI. Can I do this with JustPy?
I have already found normal widgets incl. grids, that looks quit good. But what I want to realize is for example:

- Show and manipulate data in both tables and grids.
- Subforms. I.e. show the data for selection in a table on the left, manipulate the data in a form on the right.
- Subforms for 1-n relationships. So on the left a data record and on the right the n associated data records.
- Are there also comboboxes?
- How to pack the widgets? From wxPython I know a kind of hirarchical panels, where the widgets belong to, and different kinds of geometry managers for positioning all the widgets in a window/frame. I found the Quasar Grid Gutter Element, but miss the possibility to put cells together (in HTML tables it is known as colspan and rowspan).

With thanks and greetings
Ulrich

**Comment by WolfgangFahl** [comment]
@cg-goebel Hi Ulrich - thank you for trying out justpy. As outlined in [our README ](https://justpy.io/#if-you-have-questions-suggestions-or-would-like-to-get-in-touch) stackoverflow is a much better place to ask such questions. I didn't even realize you entered this discussion issue until today - so sorry for the delay. Please use stackoverflow and I'll happily give some comments.
